main/inc/lib/message.lib.php

```diff
@@ -2894,6 +2896,15 @@ public static function getUsersThatHadConversationWithUser($userId)
                 FROM $messagesTable
                 WHERE
                     user_receiver_id = ".$userId;
```

Comment:
- Use $startDate instead of $dateStart.
- Use $endDate instead of $dateFinish (We use already "end" in many other places)
- Add Database::escape_string() 
- The same for the function getAllMessagesBetweenStudents()

Reply:
fixed!